addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -475,7 +475,7 @@ public void sendBroadcastNotification(String message, String icon, String severi
                 notificationMessage.put("severity", severity);
                 socket.emit("broadcastnotification", notificationMessage);
             } catch (JSONException e) {
-                logger.error(e.getMessage());
+                logger.error("{}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace